### DISCUSSION TITLE
We are moving to the #support channel in Slack!

### DISCUSSION THREAD

**Discussion by yemikifouly** [OpenMined/courses#405] [Announcements]
Hello! 👋

In order to answer your questions more promptly and increase the likelihood that they be answered by the members of our Engineering Teams, **WE ARE MOVING QUESTIONS TO OUR [SLACK WORKSPACE](www.slack.openmined.org)**.

As a result, we invite you, from this day forward to post all your **technical questions** related to the courses in the **#support** channel of our Slack workspace where they will be attended to by the most technically knowledgeable members of our teams.

We recommend you post your **non-technical questions** related to the course in the **#courses** channel.

_Additionally_, please post in the following channels for the mentioned uses:

🤝 **#introductions:** To introduce yourself to the community.
🤔 **#beginners:** To share beginners-related inquiries and comments.
🥸 **#general:** To start a conversation or make a comment that does not fit with the themes of the other channels.

Finally, tune in to **#announcements** to keep up with organization-wide announcements and updates.

Note that the discussion board will still be up while we transition to Slack. However, it will no longer be maintained.

Thank you for being part of our community on the discussion board. 

See you in Slack! 🙂🙌

**Comment by Jackalope2K17** [comment]
Hi, unfortunately the link to the Slack workspace does not work. Please, could you share another one?

Also, I am trying to start the last lesson of Foundations of Private Computation but I always get this message - since Friday:

> 
> Sorry, it looks like there was an error!
> 
> Our development team has been notified and is working on a fix. We apologize for the inconvenience. If you'd like to file an official bug report, [please do so here](https://github.com/OpenMined/openmined/issues).

**Reply by souravcipher** [reply]
Hi @Jackalope2K17!  You can join the slack workspace using this link - https://slack.openmined.org/

**Reply by Jackalope2K17** [reply]
@souravrhythm thank you!
